PD-2091 / 25.10 / Pd 2091 veeam smb with fast clone content (#3931)
truenas/documentation: content/Solutions/Integrations/Veeam.md

Refer to the basic [SMB Share]({{< relref "/Scale/SCALEtutorials/Shares/SMB/_index.md" >}}) instructions for information on creating a share user.
You can assign this admin user full control or a sharing admin role. Add this user to the ACL for the dataset created for Veeam SMB with fast cloning.

### Configuring TrueNAS Storage

First, think about how you want to organize your data storage. We recommend creating the parent dataset structure before creating the dataset and share for Veeam SMB fast cloning.
For example, organizing all datasets for shares (SMB, NFS, etc.) under one common parent dataset (*shares*) with datasets for each share type nested under it.

If you have other Veeam data storage set up in the system and you want to nest the fast cloning dataset under the existing Veeam dataset, skip adding the dataset organization instructions below and go to the instructions for verifying the record size for the Veeam dataset at the bottom of this section.

To create and organize datasets in a hierarchical structure for shares, for example, *shares/smb/veeam*, go to the **Datasets** screen, select the parent dataset row in the table, and click **Add Dataset**.

> Add the dataset for the next level in the organization, for example, *shares* or it exists, add *smb*, etc.

Reviewer: This isn't really making sense to me--"*shares* or it exists"? Is there a way we could rephrase this section, or maybe create a literal chart as a visual representation?

Author: It was missing a word. I've rewritten this in the next commit to make it more clear.

In many sites, Veeam compression or deduplication is around 1.5x to 2x, which is more of a reference than a rule.
Backup types, applications, and the diversity of VMs can all factor into the amount of storage you need.
You must also consider capacity alongside desired performance, as a smaller quantity of large drives often does not yield the same performance as a larger number of small drives.

> The resources listed below provide rough calculations you can use to determine the sizes and numbers of drives needed:

Reviewer: ```suggestion
The resources listed below provide rough calculations available to determine the sizes and numbers of drives needed:
```

TrueNAS is a robust, unified storage system well-suited for nearly any environment.
For backups, the platform takes advantage of the data integrity offered by ZFS, which includes features such as copy-on-write, snapshots, and checksums that prevent bit-rot.

> You can expand TrueNAS Enterprise appliances at any time by simply adding more drives, allowing datasets to grow and keep pace with your data storage needs.

Reviewer: ```suggestion
You can expand TrueNAS Enterprise appliances at any time by simply adding more drives, allowing datasets to grow and keeping pace with your data storage needs.
```
